packages/editor/src/utils/url.js

```diff
@@ -1,12 +1,8 @@
-/**
- * External dependencies
- */
-import { deburr, trim } from 'lodash';
-
 /**
  * WordPress dependencies
  */
 import { addQueryArgs } from '@wordpress/url';
+export { cleanForSlug } from '@wordpress/url';
```

Comment:
is this normal to export from another package? I've not seen it before so I'm guessing it's just new to me.

we could instead leave `cleanForSlug` in but add a deprecation notice to use the one in `@wordpress/url` instead, but still call that one

```ts
import deprecated from `@wordpress/deprecated`;

export function cleanForSlug( string ) {
	deprecated(
		'editor/utils/url::cleanForSlug replaced by url::cleanForSlug', {
		since: '12.6.0',
		plugin: 'Gutenberg',
		alternative: 'wp.url.cleanForSlug'
	} );

	return url.cleanForSlug( string );
}
```

I'm not sure how to properly indicate the `since` and `plugin` fields or how to know when `plugin` is appropriate vs. not having it there.

Reply:
> is this normal to export from another package?

Usually only when we want to alias a 3rd party lib under something like `_unstable`. I was waiting on feedback for deprecating the entire file since `getWPAdminURL` isn't particularly useful either.

I can go ahead and add a warning/changelog note for this one though.

Reply:
Updated in [ed709a3](https://github.com/WordPress/gutenberg/pull/39033/commits/ed709a34d868fcb496ffef91481c591d55d4593d)